File: server/groute/src/main/java/com/ssafy/groute/dto/PlaceReview.java
package com.ssafy.groute.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class PlaceReview {
    private int id;
    private String userId;
    private int placeId;
    private String title;
    private String content;
    private double rate;
    private String img;
    private String createDate;
}

File: client/Groute/app/src/main/java/com/ssafy/groute/src/dto/PlaceReview.kt
package com.ssafy.groute.src.dto

data class PlaceReview(
    val content: String,
    val createDate: String,
    val id: Int,
    var img: String,
    val placeId: Int,
    val rate: Double,
    val title: String,
    val userId: String
){
    constructor(placeId: Int,userId: String,content: String,rate: Double,img: String) : this(content,"",0,img,placeId, rate,"",userId)
    constructor(placeId: Int,userId: String,content: String,rate: Double,img: String,id:Int): this(content,"",id,img,placeId,rate,"",userId)
}